pkg/network/usm/monitor.go

```diff
@@ -214,7 +214,7 @@ func (m *Monitor) Start() error {
 	}
 
 	// Need to explicitly save the error in `err` so the defer function could save the startup error.
-	if m.cfg.EnableHTTPSMonitoring || m.cfg.EnableIstioMonitoring {
+	if m.cfg.EnableHTTPSMonitoring || m.cfg.EnableGoTLSSupport || m.cfg.EnableJavaTLSSupport || m.cfg.EnableIstioMonitoring {
```

Comment:
Native TLS is not a consumer of the process monitor, so why we need to check `m.cfg.EnableHTTPSMonitoring`?